src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, sans-serif;
+  color: #0000;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  text-transform: uppercase;
+  width: 100%;
+  padding-inline: 50px;
+}
+
+.is-active {
+  display: flex;
+  align-items: center;
+  color: #00acdc;
+  position: relative;
+  margin-left: auto;
+  height: 60px;
+  text-decoration: none;
+}
+
+.logo {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
```

Comment:
The 'list-style: none' style is unnecessary here because it has already been set in the '*' selector. However, since the '*' selector should be avoided, it's good to keep this declaration if you remove the '*' selector.